lib/CipherInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Aol\Transformers;
+
+interface CipherInterface
+{
+	/**
+	 * Encrypt the given text.
+	 *
+	 * @param string $plain_text The text to encrypt.
+	 *
+	 * @return string The encrypted text.
+	 */
+	function encrypt($plain_text);
```

Comment:
I don't think that's something we would want to support at this level. In the simple use case you can just `defineEncrypted(...)`. In a more complex use case where the value need to be stringified first you can handle that in the transformer itself and you have direct access to the `encrypt($plain_text)` and `decrypt($encrypted_text)` methods as you need them.